jupyter-book/trajectories/rna_velocity.ipynb

```diff
@@ -15,53 +15,117 @@
    ]
```

Comment:
<p>"M-step" Totally obvious for me what this step is, especially given the context, but might be worth spelling it out for newbies.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I think it is fine, TBH. The definition of "EM" is literally in the preceding sentence.</p>